sdk/storage/azure-storage-file/apireview/azure/storage/file/aio/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+# Stubs for azure.storage.file.aio (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._shared.models import AccountPermissions as AccountPermissions, LocationMode as LocationMode, ResourceTypes as ResourceTypes, StorageErrorCode as StorageErrorCode
+from .._shared.policies_async import ExponentialRetry as ExponentialRetry, LinearRetry as LinearRetry, NoRetry as NoRetry
+from ..models import AccessPolicy as AccessPolicy, ContentSettings as ContentSettings, CorsRule as CorsRule, DirectoryProperties as DirectoryProperties, FilePermissions as FilePermissions, FileProperties as FileProperties, Handle as Handle, Metrics as Metrics, NTFSAttributes as NTFSAttributes, RetentionPolicy as RetentionPolicy, SharePermissions as SharePermissions, ShareProperties as ShareProperties
```

Comment:
Don't see the point to expose models twice in "aio" and root level

Reply:
The reason for it is that it's convenient for user to import everything from one namespace.